src/test/scala/org/broadinstitute/dsde/rawls/webservice/RootRawlsApiServiceSpec.scala

```diff
@@ -9,6 +9,7 @@ import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport.ApplicationVersi
  */
 class RootRawlsApiServiceSpec extends ApiServiceSpec with RootRawlsApiService {
   val appVersion = ApplicationVersion("githash", "buildnumber", "version")
+  val clientSecretsJson = null
```

Comment:
if you want to keep passing the full json around, this needs to have the right json structure so that GoogleClientSecrets.load can parse it (but it should not have real secrets)